src/node/internal/internal_fs_sync.ts

```diff
@@ -0,0 +1,822 @@
+if (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any,@typescript-eslint/no-unsafe-member-access
+  !(globalThis as any).Cloudflare.compatibilityFlags['enable_node_file_system']
+) {
+  throw new Error('The experimental node:fs implementation is not enabled');
```

Comment:
Even though this is behind an experimental flag, we're now throwing a different error message, then we do in the current state. The correct way to do this is through module loader, and just act like "node:fs" doesn't exist (which is the prior behavior to this PR)